docs/source/spelling_wordlist.txt

```diff
@@ -0,0 +1,44 @@
+backend
+backends
+checkbox
+contrib
+django
+fallbacks
+flamegraph
+flatpages
+frontend
+inlining
+isort
+Jazzband
+jinja
+jQuery
+jrestclient
+js
+Makefile
+memcache
+memcached
+middleware
+middlewares
+multi
+neo
+pre
+profiler
+psycopg
+py
+pylibmc
+Pympler
+querysets
+refactoring
+spooler
+stacktrace
+stacktraces
+timeline
+Transifex
+unhashable
+uWSGI
+validator
+Werkzeug
+async
+htmx
+behaviour
+deserialized
```

Comment:
these words seem mostly copy-pasted from another project, and aren't relevant here